src/BemComponent.php

```diff
@@ -3,5 +3,6 @@
 
 interface BemComponent
 {
+  /** @return string */
```

Comment:
If you are adding a docbloc, please also put a description.

The docbloc is wasted when we have the return type already

Reply:
have added a short description to each